### DISCUSSION TITLE
默认的黑色边框怎么取消 找了好久没找到

### DISCUSSION THREAD

**Discussion by TryChai** [antvis/S2#3020] [Q&A]
![image](https://github.com/user-attachments/assets/0cd9494f-a5b5-4656-80b5-0adc70dfa08f)

**Comment by lijinke666** [comment]
https://s2.antv.antgroup.com/manual/faq#%E5%A6%82%E4%BD%95%E4%BF%AE%E6%94%B9%E9%80%89%E4%B8%AD%E6%82%AC%E5%81%9C%E5%88%B7%E9%80%89%E7%AD%89%E5%8D%95%E5%85%83%E6%A0%BC%E4%BA%A4%E4%BA%92%E4%B8%BB%E9%A2%98%E9%85%8D%E7%BD%AE

https://s2.antv.antgroup.com/api/general/s2-theme#interactionstatename
